### PR TITLE
[WIP] Payment method FAQ

### PR DESCRIPTION
**Pull request by sn3p**
**This PR can be merged when [LearningSpaces Payments](https://github.com/DefactoSoftware/LearningSpaces/pull/1479) is live.**

- [x] [English copy](https://github.com/DefactoSoftware/LearningSpaces-Landing/blob/matthijs/faq-payments/locales/en.yml#L351-L354)
- [x] [Dutch translation](https://github.com/DefactoSoftware/LearningSpaces-Landing/blob/matthijs/faq-payments/locales/nl.yml#L377-L380)
- [x] [German translation](https://github.com/DefactoSoftware/LearningSpaces-Landing/blob/matthijs/faq-payments/locales/de.yml#L365-L368)

### CODE REVIEW

File: locales/de.yml [hunk 1]
```diff
@@ -362,6 +362,11 @@ de:
               Und für noch größere Teams (wie eine Corporate Academy oder ein Unternehmen) bieten wir den individuellen Enterprise Plan.
               [Hier](/pricing.html) gibt es mehr Informationen zu unseren Plänen.
 
+          - q: What are the payment methods?
+            a: |
+              We accept payments from almost any kind of credit card (**Visa**, **MasterCard** and **American Express**).
+              We also accept payments by invoice if you have a large team. Please [contact us](javascript:Intercom('show');) and we'll be happy to help.
```
Comment (fatboypunk) [inline locales/de.yml]: Larger team or no creditcard?

File: locales/en.yml [hunk 2]
```diff
@@ -353,8 +348,10 @@ en:
               For larger teams we have the Value plan. And for really large teams (academies, large companies,
               large learning communities) we have the Enterprise plan. **For all available plans check our [Plans page](/pricing.html)**.
 
-          # - q: What are the payment methods?
-          #   a:
+          - q: What are the payment methods?
+            a: |
+              We accept payments from almost any kind of credit card (**Visa**, **MasterCard** and **American Express**).
```
Comment (fatboypunk) [inline locales/en.yml]: Larger team or no creditcard?

File: locales/nl.yml [hunk 3]
```diff
@@ -370,7 +373,12 @@ nl:
               hebben we het *Enterprise* abonnement.
 
               Kijk anders even naar [het overzicht van alle abonnementen](/pricing.html).
-          # - q: What are the payment methods?
+
+          - q: Wat zijn de betaalmethoden?
+            a: |
+              We accepteren betalingen met de meeste credit cards (**Visa**, **MasterCard** and **American Express**).
+              We accepteren ook betalingen per factuur als het om grote teams gaat. Neem dan [contact](javascript:Intercom('show');) met ons op zodat we dit kunnen regelen.
```
Comment (fatboypunk) [inline locales/nl.yml]: `Als het om grote teams gaat of er geen creditcard tot beschikking is, accepteren we ook betalingen per factuur.

File: locales/en.yml (hunk 2 above)
Comment (sn3p) [inline locales/en.yml]: What?

File: locales/nl.yml (hunk 3 above)
Comment (sn3p) [inline locales/nl.yml]: :+1:

**Pull request merged**